README.md

```diff
@@ -136,15 +136,15 @@ Please find some examples of using pe_client_tools to query the status check fac
 
 1. To find the complete output of pe_status_check from all nodes listed by certname:
 
-        `puppet query 'facts[certname,value] { name = "pe_status_check" }'`
-
+      ```puppet query 'facts[certname,value] { name = "pe_status_check" }'```
```

Comment:
I am not sure this will work in github suggestions, but lets see.
```suggestion
```shell
puppet query 'facts[certname,value] { name = "pe_status_check" }'
```
```

Reply:
Nope. Its missing the bottom ```

Reply:
gotcha